modules/k8s/jsonpath_test.go

```diff
@@ -0,0 +1,60 @@
+package k8s
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUnmarshalJSONPath(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name        string
+		jsonBlob    []byte
```

Comment:
NIT: Could make this a `string` and have the test code convert to a byte array.

Reply:
Updated 0dab3d7